projects/optic/src/commands/oas/shapes/streams/documented-bodies.ts

```diff
@@ -264,7 +263,7 @@ async function decodeCapturedBody(
     try {
       value = await CapturedBody.json(capturedBody);
     } catch (err) {
-      SentryClient.captureException(err);
+      logger.error(err);
```

Comment:
I think this should be in the responsibility of the user to fix this here - changing this to log error instead of sentry error.

This could be a malformed har file, or postman file that causes this error. Captured data / interactions here should be correctly formed from our code (if there's a mismatch, this is something an integration test should be catching this)